test/TrackingCollectionTests/TrackingCollectionTests.cs

```diff
@@ -14,6 +14,7 @@
 using System.Threading.Tasks;
 using System.Reactive.Threading.Tasks;
 using GitHub;
+using System.Globalization;
 
 [TestFixture]
```

Comment:
I'm not sure there is any value in continuing to run these flaky tests.

```suggestion
[TestFixture, Ignore("These tests are flaky and we're planning to move from using TrackingCollection")]
```